drivers/include/periph/i2c.h

```diff
@@ -11,12 +11,20 @@
  * @ingroup     drivers_periph
  * @brief       Low-level I2C peripheral driver
  *
- * @{
- * @file
- * @brief       Low-level I2C peripheral driver interface definition
+ * This is a simple I2C interface to allow platform independent access to a
+ * MCU's I2C bus and peripherals. This interface is intentionally designed as
+ * simple as possible, to allow for easy implementation and maximal portability.
+ *
+ * @note        The current version of this interface only supports the
+ *              7-bit addressing mode.
+ *
+ * @note        This interface is due for remodeling, hence API changes are to
+ *              be expected for upcoming releases.
+ *
+ * @section     sec_debug A note on pull-up resistors for SDA/SCL
```

Comment:
I think you're right ... 

to my defense, I copied from [pwm.h](https://github.com/RIOT-OS/RIOT/blob/master/drivers/include/periph/pwm.h#L41). Should I fix this anyway (and pwm, too)??